app/controllers/application_controller.rb

```diff
@@ -8,4 +8,14 @@ def authenticate_user_is_admin
       redirect_to root_path, alert: "You are not authorized to access that!"
     end
   end
+
+  def after_sign_in_path_for(user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `after_sign_in_path_for(*)` if you want the method to accept any arguments but don't care about them.